concepts/arrays/about.md

```diff
@@ -0,0 +1,149 @@
+# About
+
+JSON defines an array as:
+
+> An array is an ordered collection of values.
+> An array begins with `[` left bracket and ends with `]` right bracket.
+> Values are separated by `,` comma.
+
+Arrays can contain zero or more of _any kind_ of JSON values.
+Array elements do not need to be all the same type.
+
+## Creating arrays
+
+Use brackets to collect elements into an array
```

Comment:
Note, the sentences prior to example codeblocks end in: nothing (here), periods (line 19), and colons (line 33).

I don't know how much you care about being consistent about that :)